src/Humanizer.Tests/Localisation/DefaultFormatterTests.cs

```diff
@@ -20,5 +21,25 @@ public void TimeSpanHumanizeThrowsExceptionForTimeUnitsLargerThanWeek(TimeUnit t
         {
             Assert.Throws<ArgumentOutOfRangeException>(() => new DefaultFormatter(CultureInfo.InvariantCulture.Name).TimeSpanHumanize(timeUnit, unit));
         }
+
+        [Fact]
+        public void Issue_392_A_collection_formatter_for_the_current_culture_has_not_been_implemented_yet()
```

Comment:
I see where you're going with the method name; but it's inconsistent with the rest of the codebase. Please rename it to something like `HandlesNotImplementedCollectionFormattersGracefully`. You can provide an actual url to the issue on GitHub in a comment.